src/Model/Restriction.php

```diff
@@ -269,33 +280,38 @@ protected function cancelBookings( $bookings ) {
 	}
 
 	/**
-	 * Send mails regarding item/location admins and booked timeslots.
+	 * Send restriction mails regarding item/location admins and booked timeslots.
 	 *
-	 * @param Booking[]
+	 * @param Booking[] $bookings booking post objects.
 	 */
 	protected function sendRestrictionMails( $bookings ) {
 		$userIds = [];
 
 		foreach ( $bookings as $booking ) {
 			// User IDs from booking
 			$userIds[] = $booking->getUserData()->ID;
+        }
 
-			foreach ( $userIds as $userId ) {
-				$hintMail = new RestrictionMessage( $this, get_userdata( $userId ), $booking, $this->getType() );
-				$hintMail->triggerMail();
-			}
-		}
-	}
+        // Delete duplicate user-ids so that restriction mail will only be send once to each user
+        $userIds = array_unique( $userIds );
+
+        foreach ( $userIds as $userId ) {
```

Comment:
@hansmorb : Das wird so nicht funktionieren, weil so nur die letzte Buchung mit einer Mail informiert wird. 
@chriwen : Was soll denn erreicht werden? Im Grunde muss ja jeder User für jede seiner Buchungen eine Info bekommen...

Reply:
@markus-mw Soweit ich das verstehe, versendet dieser Teil ja die Info das Artikel X im Zeitraum X-Y nicht mehr buchbar ist. Wenn nun Nutzer A den Artikel mehrmals in diesem Zeitraum gebucht hat, reicht hier diese Info einmal. Die Stornierungsmails der einzelnen Buchungen werden ja an anderer Stelle generiert und für jede Buchung dann auch eine Stornierungsmail versendet. Das ist auch richtig. Nur die Info-Mail über den Ausfall würde in dem Fall einmalig reichen. Ich dachte, dies wäre mit der obigen Lösung realisiert.